# This file is distributed under the same license as the Django package.
#
# Translators:
# alkuma <alok.kumar@gmail.com>, 2013
# Chandan kumar <chandankumar.093047@gmail.com>, 2012
# Jannis Leidel <jannis@leidel.info>, 2011
# Sandeep Satavlekar <sandysat@gmail.com>, 2011
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-07-24 22:05+0300\n"
"PO-Revision-Date: 2017-09-24 14:24+0000\n"
"Last-Translator: Jannis Leidel <jannis@leidel.info>\n"
"Language-Team: Hindi (http://www.transifex.com/django/django/language/hi/)\n"
"Language: hi\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr "व्यक्तिगत सूचना"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr "अनुमतियाँ"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:50
msgid "Important dates"
msgstr "महत्त्वपूर्ण तिथियाँ"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:134
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:144
msgid "Password changed successfully."
msgstr "शब्दकूट बदली कामयाब"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:164
#, python-format
msgid "Change password: %s"
msgstr "शब्दकूट बदलें: %s"

#: env/lib/python3.8/site-packages/django/contrib/auth/apps.py:15
msgid "Authentication and Authorization"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/base_user.py:48
msgid "password"
msgstr "शब्दकूट"

#: env/lib/python3.8/site-packages/django/contrib/auth/base_user.py:49
msgid "last login"
msgstr "पिछला लॉगिन"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:40
msgid "No password set."
msgstr "कोई कूटशब्द नहीं।"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:45
msgid "Invalid password format or unknown hashing algorithm."
msgstr "अवैध कूटशब्द प्रारूप या अज्ञात द्रुतान्वेषण कलन विधि"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:87
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:334
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:407
#, fuzzy
#| msgid "The two password fields didn't match."
msgid "The two password fields didn’t match."
msgstr "यह दो शब्दकूट क्षेत्रों का मेल नहीं होता "

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:90
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:143
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:179
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:411
msgid "Password"
msgstr "कूटशब्द"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:96
msgid "Password confirmation"
msgstr "कूटशब्द पुष्टि"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:99
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:420
msgid "Enter the same password as before, for verification."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:145
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:186
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr "कृपया सही %(username)s व कूटशब्द भरें। भरते समय लघु और दीर्घ अक्षरों का ध्यान रखें।"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:189
msgid "This account is inactive."
msgstr "यस खाता सुस्त है"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:252
msgid "Email"
msgstr "डाक पता"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:337
msgid "New password"
msgstr "नया शब्दकूट"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:343
msgid "New password confirmation"
msgstr "नया शब्दकूट पुष्टि"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:379
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "आपने पुराना शब्दकूट गलत दर्ज किया है । कृपया फिर से दर्ज करें"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:382
msgid "Old password"
msgstr "पुराना शब्दकूट"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:417
msgid "Password (again)"
msgstr "शब्दकूट (दुबारा)"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:259
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:333
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:429
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:489
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:520
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:556
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:592
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:630
msgid "algorithm"
msgstr "अलगोरिथम"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:260
msgid "iterations"
msgstr "पुनरूक्तियाँ"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:261
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:339
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:431
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:490
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:521
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:631
msgid "salt"
msgstr "साल्ट"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:262
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:340
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:491
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:522
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:557
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:593
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:632
msgid "hash"
msgstr "हैश"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:334
msgid "variety"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:335
msgid "version"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:336
msgid "memory cost"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:337
msgid "time cost"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:338
msgid "parallelism"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:430
msgid "work factor"
msgstr "कार्य फ़ैक्टर"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:432
msgid "checksum"
msgstr "चेकसम"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:56
#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:108
msgid "name"
msgstr "नाम"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:60
msgid "content type"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:62
msgid "codename"
msgstr "कोडनेम"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:67
msgid "permission"
msgstr "अनुमति"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:68
#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:111
msgid "permissions"
msgstr "अनुमतियाँ"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:118
msgid "group"
msgstr "वर्ग"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:119
#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:242
msgid "groups"
msgstr "वर्गों"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:233
msgid "superuser status"
msgstr "सर्वोच्च प्रयोक्ता स्थिति"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:236
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""
"निर्दिष्ट करता है कि जो इस उपयोगकर्ता के पास सभी अनुमतियाँ उन्हें बिना बताए स्पष्ट रूप से "
"निर्धारित है."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:245
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:253
msgid "user permissions"
msgstr "प्रयोक्ता अनुमतियाँ"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:255
msgid "Specific permissions for this user."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:326
msgid "username"
msgstr "प्रयोक्ता नाम"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:329
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:332
msgid "A user with that username already exists."
msgstr "इस नाम के साथ प्रवोक्ता अस्तित्व है"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:335
msgid "first name"
msgstr "पहला नाम"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:336
msgid "last name"
msgstr "आखिरी नाम"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:337
msgid "email address"
msgstr "डाक पता"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:339
msgid "staff status"
msgstr "कर्मचारी स्थिति"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:341
msgid "Designates whether the user can log into this admin site."
msgstr "तय करता हैं की उपयोगकर्ता इस साईट प्रशासन में प्रवेश कर सकता हैं या नहीं |"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:344
msgid "active"
msgstr "सक्रिय"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:347
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"निर्दिष्ट करता है कि क्या इस उपयोगकर्ता को सक्रिय माना जाना चाहिए.खातों को हटाने की  "
"बजाय इस अचयनित करे."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:351
msgid "date joined"
msgstr "तिथि भरती"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:360
msgid "user"
msgstr "उपभोक्ता"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:361
msgid "users"
msgstr "उपभोक्ताऐं"

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
msgstr[1] ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] ""
msgstr[1] ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:157
msgid "Your password can’t be too similar to your other personal information."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:183
msgid "This password is too common."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:188
msgid "Your password can’t be a commonly used password."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:198
msgid "This password is entirely numeric."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:203
msgid "Your password can’t be entirely numeric."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "%(site_name)s पर कूटशब्द को पुनःठीक करे"

#: env/lib/python3.8/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:160
msgid "Logged out"
msgstr "लाग्ड आउट "

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:217
msgid "Password reset"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:244
msgid "Password reset sent"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:254
msgid "Enter new password"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:314
msgid "Password reset unsuccessful"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:322
msgid "Password reset complete"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:334
msgid "Password change"
msgstr ""

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:357
msgid "Password change successful"
msgstr ""
